app/views/campaigns/_settings_modal.html.erb

```diff
@@ -4,7 +4,7 @@
 
 	<%= render 'common/modal_header', title: 'Campaign Settings' %>
 
-  <form class='form--flatFields' autosubmit action='/nonprofits/<%=@nonprofit.id%>/campaigns/<%=@campaign.id%>' method='put' data-reload-with-slug parsley-validate>
+  <form class='form--flatFields' autosubmit action='<%= campaigns_path(@campaign.nonprofit, @campaign) %>' method='put' data-reload-with-slug parsley-validate>
```

Comment:
We should probably have a custom route helper that allows you to do something like `alternate_campaigns_path(@campaign)` because the campaign object in `@campaign` actually provides enough information to generate the route.